src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,123 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { useEffect, useRef, useState } from 'react';
+
+type Props = {
+  todo: Todo;
+  onDelete: (val: number) => void;
```

Comment:
To onDelete ale tez pewnie inne takie funkcje przekazujesz najpierw z App do TodoList a pozniej do TodoItem. Tak na prawde funkcji usówania używasz tylko i wyłacznie w TodoItem wiec wydaje mi sie ze mogłabys po prostu uzyc tego tutaj i nie mieszac w to TodoList oraz App. Oni nie musza o tym wiedziec ;))